crates/aranya-crypto/src/afc/mod.rs

```diff
@@ -32,12 +32,12 @@ pub use uni::*;
 use crate::error::Error;
 
 // This is different from the rest of the `crypto` API in that it
-// allows users to directly access key material (`ChannelKeys`,
+// allows devices to directly access key material (`ChannelKeys`,
```

Comment:
revert this change: "users" means "users of `aranya-crypto`," i.e., the people writing the code.